charts/common/values.schema.json

```diff
@@ -619,43 +619,89 @@
                 }
               }
             },
-            "networkpolicy": {
+            "networkpolicies": {
               "type": "object",
-              "required": [
-                "deploy"
-              ],
-              "properties": {
-                "deploy": {
-                  "type": "boolean",
-                  "default": false
-                },
-                "ipBlock": {
+              "patternProperties": {
+                "^.*$": {
                   "type": "object",
-                  "required": [
-                    "cidr"
-                  ],
                   "properties": {
-                    "cidr": {
-                      "type": "string"
-                    }
-                  }
-                },
-                "ingressPorts": {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "properties": {
-                      "protocol": {
-                        "type": "string",
-                        "default": "TCP",
-                        "enum": [
-                          "TCP",
-                          "UDP",
-                          "SCTP"
-                        ]
-                      },
-                      "port": {
-                        "type": "integer"
+                    "policyTypes": {
+                      "type": "array"
+                    },
+                    "ingress": {
+                      "type": "object",
+                      "properties": {
+                        "ipBlock": {
+                          "type": "object",
+                          "required": [
+                            "cidr"
+                          ],
+                          "properties": {
+                            "cidr": {
+                              "type": "string"
+                            }
+                          }
+                        },
+                        "ports": {
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "required": [
+                              "protocol",
+                              "port"
+                            ],
+                            "properties": {
+                              "protocol": {
+                                "type": "string",
+                                "default": "TCP",
+                                "enum": [
+                                  "TCP",
+                                  "UDP",
+                                  "SCTP"
+                                ]
+                              },
+                              "port": {
+                                "type": "integer"
```

Comment:
Can according to the API also be a string:
```
   port	<string>
     The port on the given protocol. This can either be a numerical or named
     port on a pod. If this field is not provided, this matches all port names
     and numbers. If present, only traffic on the specified protocol AND port
     will be matched.
```

Better to set type as follows:
```
"type": [
  "number",
  "string"
]
```